### DISCUSSION TITLE
Plotting "sphere meridian" in PF

### DISCUSSION THREAD

**Discussion by GitHubUserWithoutAName** [mtex-toolbox/mtex#953] [Ask Anything] [answered]
Dear MTEX community,

is it possible to plot the different "meridians" of the unit sphere into an equal-angle pole figure plot (i.e. the "vertical" lines of a Wulff net, spaced regularly in terms of the angle)? I have played around with traces of various planes and different fibres, but to no avail. Plotting the "horizontal" parts of a Wulff net is easily possible using fibres.

Is there maybe even a predefined way of plotting a Wulff net similar to the "grid" functionality that PF plots offer? That would be really practical.

Thanks for any help!

**Comment by kilir** [comment, marked as answer]
Hi,
do you mean something like that:
```
f = newMtexFigure
sp = newSphericalPlot(f.currentAxes,'upper','grid','on','grid_res',90*degree)

```

Cheers,
Rüdiger

**Reply by GitHubUserWithoutAName** [reply]
Hi Rüdiger, 
thanks for your swift response! 

No, I meant part of the lines comprising a Wullf net, cf. e.g. Wikipedia: [Wulff net](https://de.wikipedia.org/wiki/Wulffsches_Netz). What I meant with vertical lines are those lines that are obainted by projecting the meridians of the sphere stereographically (and which are somewhat vertical as a general direction, i.e. from top to bottom as opposed to horizontal).

**Reply by kilir** [reply]
You can draw any great circle by adding the option `'plane'` to `vector3d/plot`. 
```
v = vector3d.byPolar([0:10:180]*degree,0,'antipodal')
plot(v,'plane','projection','eangle')
```
Cheers,
Rüdiger

**Reply by GitHubUserWithoutAName** [reply]
Perfect, thanks!
